modules/migration-cleaning-up-failed-migration.adoc

```diff
@@ -0,0 +1,132 @@
+
+
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration-toolkit-for-containers/troubleshooting-mtc
+
+
+:_module-type: PROCEDURE
+
+[id="cleanup-of-failed-migration_{context}"]
+= Cleaning up a failed migration
+
+.Prerequisites
+* You must be logged in to the {product-title} cluster as a user with the `cluster-admin` role.
+* You must have the OpenShift CLI installed.
+
+
+.Procedure
+
+
+. Deleting resources
++
+If a migration fails during Stage or Copy, some resources are retained to allow debugging and need to be cleaned up.
++
+The following resources must be deleted:
+
+* rsync/stunnel pods, both in the source and target namespaces
+* Secrets and configmap, both in the source and target namespaces
+* Route and service only in the target namespaces
+
+
+
+
+. Unquiescing applications
++
+If your application was quiesced during migration, you must unquiesce it by scaling it back to its initial replica count. The original replica count is indicated in a label that MTC adds to the `deployment` resource when a source application is quiesced during migration:
+
++
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+ annotations:
+    deployment.kubernetes.io/revision: "1"
+    migration.openshift.io/preQuiesceReplicas: "1"
+----
+Unquiesce the application in either of the following two ways:
+
+** Manually edit the deployment primitive (`Deployment`, `DeploymentConfig`, etc.) and set the `spec.replicas` field back to its original, non-zero value:
++
+----
+$ oc edit deployment <deployment_name>
+----
+
+** Scale your deployment with the `oc scale` command:
++
+----
+$ oc scale deployment <deployment_name> --replicas=<desired_replicas>
+----
+
+
+. Deleting the MTC Operator and cluster-scoped resources
+
++
+CAUTION: This procedure entirely removes Velero. Perform it only if there are no more Velero instances employed on the cluster.
```

Comment:
We do not mark up callouts this way. Also, I am fairly certain we do not use "CAUTION".
How is the user supposed to know whether there are no more Velero instances?

Employed? Not deployed?